webpack/webpack.config.dev.js

```diff
@@ -2,6 +2,9 @@ const webpack = require('webpack');
 
 const sharedConfig = require('./webpack.config.shared.js');
 
+const remotedev = require('remotedev-server');
+remotedev({ hostname: 'localhost', port: 9090 });
```

Comment:
As I don't think starting this server has anything to do with configuring webpack, I'd remove it from here and probably use the CLI to start it instead. (Similar to how the webpack dev server is started.)